crates/cext/src/transpiler/target.rs

```diff
@@ -0,0 +1,1418 @@
+// This code is part of Qiskit.
+//
+// (C) Copyright IBM 2025
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE.txt file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+use crate::exit_codes::ExitCode;
+use crate::pointers::{const_ptr_as_ref, mut_ptr_as_ref};
+use indexmap::IndexMap;
+use qiskit_accelerate::{
+    nlayout::PhysicalQubit,
+    target_transpiler::{InstructionProperties, Qargs, Target},
+};
+use qiskit_circuit::operations::{Operation, Param, StandardGate};
+use std::ffi::{c_char, CStr, CString};
+use std::mem::forget;
+use std::ptr::null_mut;
+
+/// @ingroup QkTarget
+/// Construct a new Target with the given number of qubits.
```

Comment:
It would be nice to consistently have 
```
``Target``
```
(or double quotes for other args/code words 🙂)

Reply:
Fixed in 6ac8072